challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+  input.map{|array_item| array_item.capitalize}
```

Comment:
`array_item` is a bit cumbersome.  `item` is better, and `i` is okay when it's short for 'item'.  But better than any of that is to provide more of a reminder about what's in there, since the array items are strings, `str` is probably the best choice.